oauth2_http/java/com/google/auth/oauth2/ExternalAccountSupplierContext.java

```diff
@@ -71,6 +74,18 @@ public Builder setSubjectTokenType(String subjectTokenType) {
       return this;
     }
 
+    /**
+     * Sets the subject token type.
```

Comment:
You forgot to remove the other setSubjectTokenType?

Reply:
We have both in the credential builders as well - the enum is just to make it easier to build from code. The credential doesn't actually have it stored as an enum, just as a string.